### PR TITLE
Updated with Bind information

### PR DESCRIPTION
**Pull request by ravigadde**

### CODE REVIEW

**Comment by ravigadde** [comment]
cc @timothysc

File: contributors/design-proposals/scheduler_extender.md [hunk 1]
```diff
@@ -31,6 +34,9 @@ type ExtenderConfig struct {
 	FilterVerb string `json:"filterVerb,omitempty"`
 	// Verb for the prioritize call, empty if not supported. This verb is appended to the URLPrefix when issuing the prioritize call to extender.
 	PrioritizeVerb string `json:"prioritizeVerb,omitempty"`
+	// Verb for the bind call, empty if not supported. This verb is appended to the URLPrefix when issuing the bind call to extender.
+	// If this method is implemented by the extender, it is the extender's responsibility to bind the pod to apiserver.
+	BindVerb string
```
Comment (k82cn) [inline contributors/design-proposals/scheduler_extender.md]: metadata?

**Comment by timothysc** [comment]
Pending on https://github.com/kubernetes/kubernetes/pull/41447

**Comment by ravigadde** [comment]
@k82cn Updated this. Could this PR be merged?

**Comment by k82cn** [comment]
Please also highlight that only one `bind` should be implemented.

Others are lgtm :).

**Pull request merged**